CONTRIBUTING.md

```diff
@@ -2,10 +2,12 @@
 
 - [Contributing to PyTorch](#contributing-to-pytorch)
 - [Developing PyTorch](#developing-pytorch)
-  - [Nightly Checkout & Pull](#nightly-checkout--pull)
+  - [Tips and Debugging](#tips-and-debugging)
```

Comment:
wondering why this is not auto-generated :-(

Reply:
the only reason I noticed it was out of sync is because VS Code auto-generated it while I was writing this PR ;)